scripts/Elemental_Analysis.py

```diff
@@ -43,6 +66,21 @@ def select_data_file(self):
         if filename:
             self.ptable.set_peak_datafile(filename)
 
+    def add_plot(self):
+        name = "Plot {}".format(time())
+        subplot = self.plotting.add_subplot(name)
+        self.plotting.call_plot_method(name, subplot.set_title, name)
+        plot1 = mantid.CreateSampleWorkspace(OutputWorkspace=str(time()))
+        self.plotting.plot(name, plot1)
+        plot2 = mantid.Plus(plot1, plot1, OutputWorkspace=str(time()))
+        self.plotting.plot(name, plot2)
+        self.plotting.add_hline(name, 0.06, 0, 1)
+        self.plotting.add_vline(name, 10100, 0, 1)
+
+    def del_plot(self):
+        to_del = random.choice(self.plotting.get_subplots().keys())
+        self.plotting.remove_subplot(to_del)
+
```

Comment:
not tried it yet but I suspect if you close your GUI main window the plot will hang around. You can stop this by overloading the closeEvent method and manually closing the plot

Reply:
Deliberate for now - I'll be creating a separate issue for this when docking comes about